.fixtures.yml

```diff
@@ -0,0 +1,8 @@
+fixtures:
+  forge_modules:
```

Comment:
can you please update this to use the github branches? We normally test against against the master branches within spec tests

Reply:
Changed.